src/PermissionServiceProvider.php

```diff
@@ -41,9 +41,12 @@ public function boot()
 
         $this->registerModelBindings();
 
-        $this->app->afterResolving(Gate::class, function () {
+        try {
+            DB::connection()->getPdo();
             app(PermissionRegistrar::class)->registerPermissions();
-        });
+        } catch (\Exception $e) {
+            Log::error("Could not connect to the database.  Please check your configuration. error:" . $e);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Avoid using static access to class '\Illuminate\Support\Facades\Log' in method 'boot'.](https://app.codacy.com/manual/mostafamaklad/laravel-permission-mongodb/pullRequest?prid=5194487)